### ISSUE TITLE
Migrating GitHub integrations to newer APIs

### ISSUE THREAD

**Issue by RoiEXLab** [triplea-game/triplea#4825]
I just realized there have been updates to travis-ci and lgtm, that use the new github check API and are slightly better integrated into the whole GitHub ecosystem.

### LGTM
https://discuss.lgtm.com/t/announcing-migration-to-github-apps/2052
For LGTM it looks like we simply need to install the App for this repository, and everything should work out of the box. This way the alerts are no longer posted using @ron-murhammer's account + several other improvements.

### Travis-CI
It's a little bit trickier for that one, however it looks like the migration process is well-documented:
https://docs.travis-ci.com/user/migrate/open-source-repository-migration#migrating-a-repository
Basically after migrating to travis-ci.com (from travis-ci.org) travis will use the check api from then forward.

Might be worth to do that for all repositories for triplea-game to be consistent, but only @ron-murhammer can do that unfortunately.

**Comment by ron-murhammer** [comment]
@RoiEXLab Yeah, good find. I'll try to take a look at these over the next week or so.

**Comment by ron-murhammer** [comment]
LGTM should now be updated to use the app and enabled on all triple-game repos.
